src/qae/object-definition.js

```diff
@@ -481,6 +492,24 @@ export default objectDefinition;
  * @property {string|StringExpression} [label]
  */
 
+/**
+ * @typedef {object} trendLine
+ * @property {string} label Trend line label
+ * @property {string} qType Trend line type, for example LINEAR or EXPONENTIAL
+ * @property {boolean} flipXY=false Set to true if the first measure should be the dependent variable
+ * @property {string} qMultiDimMode="Sum" Determines if you get one trendline of each value in the other dimension or trendline based on the sum of the value in the other dimension
```

Comment:
I don't think the value of qMultiDimMode makes any difference for scatterplot maybe update the comment to reflect that.